include/dronecore.h

```diff
@@ -30,17 +59,83 @@ class DroneCore
         CONNECTIONS_EXHAUSTED
     };
 
+    /**
+     * Translates the ConnectionResult enum to a human-readable English string.
+     */
     static const char *connection_result_str(ConnectionResult);
 
+    /**
+     * Adds a UDP connection with the default arguments.
+     *
+     * This will listen on UDP port 14540 on localhost.
+     *
+     * @return the result of adding the connection.
+     */
     ConnectionResult add_udp_connection();
+
+    /**
+     * Adds a UDP connection with a specific port number.
+     *
+     * @param local_port_number The local UDP port to listen to
+     * @return the result of adding the connection.
+     */
     ConnectionResult add_udp_connection(int local_port_number);
 
+    /**
+     * Return vector of device UUIDs.
+     *
+     * This returns a vector of the UUIS of all devices that have been discovered so far and can be
+     * accessed.
+     *
+     * @return a reference to the vector containing the UUIDs
+     */
     const std::vector<uint64_t> &device_uuids() const;
+
+    /**
+     * Return the only discovered device.
+     *
+     * This returns the only discovered device or a null device if no device has been discovered
+     * yet. If there is more than one device discovered, it will just return the device which was
+     * discovered first.
+     *
+     * @return a reference to a device
+     */
     Device &device() const;
+
+    /**
+     * Return a device given a UUID.
```

Comment:
`Returns the device with the specified UUID.`

Reply:
Fixed in 8327f85.